Database/Corrections/cataQuestFixes.lua

```diff
@@ -1644,6 +1644,9 @@ function CataQuestFixes.Load()
             [questKeys.objectives] = {},
             [questKeys.triggerEnd] = {"Fire Trial Completed",{[zoneIDs.AZSHARA] = {{33.43,23.39}}}},
         },
+        [14304] = { -- Blood Theory
+            [questKeys.objectives] = {{{25412,nil,Questie.ICON_TYPE_INTERACT},nil,{{49136}},nil}},
```

Comment:
This does not work.

It is important, that the count of `objectives` in the Questie DB matches, the actual amount of objectives given by `/dump C_QuestLog.GetQuestObjectives(14304)`. The API says there is only a single objective of type "item". And that is already correct in the DB.

What we need instead is an `extraObjective` to also point to NPC 25412

Reply:
https://github.com/Questie/Questie/pull/6179/commits/07b43e9d57f77ce52ec0ca9f4297bb7158022a50